test/units/testsuite-53.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+# This test makes some basic checks that RPM macros work correctly.
+# RPM is a simple C program available on different Linux distros, not only RPM-based ones,
+# and even BSD systems, so it must not be a problem to require it.
+# rpmspec utility is required (so this test will work with RPM 4 but won't work with RPM 5).
+set -xefu
```

Comment:
The `set -f` to disable globbing is a bit unusual... I don't really see that often. Why did you decide to use it here?

Reply:
Actually -f makes nothing here, that is just my habbit to use it to improve predictability of results...